pkg/aggregator/demultiplexer_agent.go

```diff
@@ -129,7 +129,8 @@ func InitAndStartAgentDemultiplexer(
 	haAgent haagent.Component,
 	compressor compression.Component,
 	tagger tagger.Component,
-	hostname string) *AgentDemultiplexer {
+	hostname string,
+) *AgentDemultiplexer {
```

Comment:
Please do not reformat code that didn't otherwise change.

Reply:
done

Reply:
This was just one example, there are a lot of unrelated style changes in your PR.